lib/msf/core/auxiliary/web/http.rb

```diff
@@ -110,7 +110,7 @@ def connect
       opts[:target].port,
       {},
       opts[:target].ssl,
-      'SSLv23',
+      'Auto',
```

Comment:
Any reason this is not `@opts[:ssl_version]` like in `lib/anemone/rex_http.rb`?

Reply:
Notice this object expects `opts[:target]` to contain the targeting details, [lib/msf/core/auxiliary/web/target.rb](https://github.com/rapid7/metasploit-framework/blob/ff7b017e147a87b5d6a5d8a5c0a1c7cd257aebe8/lib/msf/core/auxiliary/web/target.rb) does not have an `ssl_version` option access, by using `Auto` this provide the most compatibility with whatever OpenSSL version Ruby provides.

Reply:
Makes sense to me, considering this resolved, though we probably aught to come back to this at some point and look at how we can make this fall in line with the `ssl_version` standard code other modules and libraries are using. I'll leave this open for passerby's in case we do decide to make a run through this at some point, might be easier for them to find this when searching for code cases.

For those interested, Jeffery also noted that `Auxiliary::Web::Target` is not actually used in the framework directly and is actually used in Pro code which might be part of the reason for this discrepancy.